containers/src/Kokkos_DualView.hpp

```diff
@@ -1050,6 +1053,37 @@ class DualView : public ViewTraits<DataType, Arg1Type, Arg2Type, Arg3Type> {
         /* Mark Host copy as modified */
         ++modified_flags(0);
       }
+    };
+
+    constexpr bool has_execution_space = alloc_prop_input::has_execution_space;
+
+    if constexpr (has_execution_space) {
+      using ExecSpace = typename alloc_prop_input::execution_space;
+      const auto& exec_space =
+          Impl::get_property<Impl::ExecutionSpaceTag>(arg_prop);
```

Comment:
Should `Impl:: ...` be user-facing (assuming that `Kokkos_DualView.hpp` is a public header)?  Also, should it be `(arg_prop)` or `(properties)`, as in line 1044 above?

Reply:
No, `properties` is an argument to the generic lambda. This is not inside the lambda.